lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -349,19 +349,22 @@ def set_box_aspect(self, aspect, *, zoom=1):
         aspect : 3-tuple of floats or None
             Changes the physical dimensions of the Axes3D, such that the ratio
             of the axis lengths in display units is x:y:z.
+            If None, defaults to (4,4,3).
 
-            If None, defaults to 4:4:3
-
-        zoom : float
-            Control overall size of the Axes3D in the figure.
+        zoom : float, default: 1
+            Control overall size of the Axes3D in the figure. Must be > 0.
         """
+        if zoom <= 0:
+            raise ValueError(f'Argument zoom = {zoom} must be > 0')
+        self.zoom = zoom
```

Comment:
This should be reverted as well. We don't have the attribute for now.

Reply:
I'd like to keep `self.zoom` as an attribute that so it can be read. `set_box_aspect` is called during the Axes3D init, so it'll always be there.

Reply:
Made this a private attr, think that should be good